datadog_checks_dev/datadog_checks/dev/plugin/tox.py

```diff
@@ -27,7 +27,13 @@
 # TODO: remove extra when we drop Python 2
 MYPY_DEP = 'mypy[python2]==0.910'
 # TODO: when we drop Python 2 and replace with --install-types --non-interactive
-TYPES_DEPS = ['types-PyYAML==5.4.10', 'types-python-dateutil==2.8.2', 'types_requests==2.25.11', 'types_six==1.16.2']
+TYPES_DEPS = [
+    'types-PyYAML==5.4.10',
+    'types-python-dateutil==2.8.2',
+    'types_requests==2.25.11',
+    'types_six==1.16.2',
+    "types-simplejson==3.17.5",
```

Comment:
```suggestion
    'types-simplejson==3.17.5',
```
very small style nit